qml/js/yleApi.js

```diff
@@ -28,12 +28,57 @@ var filterAvailablePrograms = function(programs) {
     }, []);
 }
 
+var zeropad = function(str, size) {
+    var s = str;
+    while (s.length < size) s = "0" + s;

```

Comment:
I'd prefer always to use curly braces after while, if.

Objections? :)

Reply:
I actually also prefer that way, this was taken from my old work-in-progress code so I didn't yet check all of the formatting.

Reply:
:+1: